### DISCUSSION TITLE
What file do you drag and drop into the coordinates reader?

### DISCUSSION THREAD

**Discussion by xanderb730** [MoffKalast/Sailwind-Map#22] [Q&A]
I found the path on my computer, but it contains 3 files and when i import them everything reads either zero or NaN.

**Comment by MoffKalast** [comment]
There's [a short rundown in the contributing guide that might help](https://github.com/MoffKalast/Sailwind-Map/blob/main/Contributing.md#updating-layer-data), though that's for tracing islands specifically but the process is the same.

The gist of it is that there should be a file called `coords_0.txt` in 
`C:\Program Files (x86)\Steam\steamapps\common\Sailwind\Mods\CoordinateViewer` if you're on Windows.
